spec/cohort-spec.js

```diff
@@ -0,0 +1,80 @@
+import Manager from '../src/FileManager.js'
+import Cohort from '../src/Cohort.js'
+import Student from '../src/Student.js'
+
+describe('Manager', () => {
+  let manager
+
+  beforeEach(() => {
+    manager = new Manager()
+  })
+
+  it('To add a cohort by name', () => {
+    const cohort = new Cohort('Cohort 1')
+    manager.addCohort(cohort)
+    expect(manager.cohorts.includes(cohort)).toBe(true)
+  })
+
+  it('To find a cohort by name', () => {
+    const cohort = new Cohort('Cohort 1')
+    manager.addCohort(cohort)
+    const found = manager.findCohortByName('Cohort 1')
+    expect(found).toBe(cohort)
+  })
+
+  it('To throw an error when trying to find a non-existent cohort', () => {
+    expect(() => {
+      manager.findCohortByName('NonExistentCohort')
+    }).toThrowError('This cohort does not exist')
+  })
+
+  it('To remove a cohort by name', () => {
+    const cohort1 = new Cohort('Cohort 1')
+    const cohort2 = new Cohort('Cohort 2')
+    manager.addCohort(cohort1)
+    manager.addCohort(cohort2)
+    manager.removeCohortByName('Cohort 1')
+    expect(manager.cohorts.includes(cohort1)).toBe(true)
+    expect(manager.cohorts.includes(cohort2)).toBe(true)
```

Comment:
This test is giving you a false positive.

`expect(manager.cohorts.includes(cohort1)).toBe(true)`
this should be:
`expect(manager.cohorts.includes(cohort1)).toBe(false)`